src/js/components/Anchor/doc.js

```diff
@@ -53,6 +53,10 @@ or just use children.`,
 this component. But, it can be adjusted directly via this size property, typically
 when it is not contained in a 'Heading', 'Paragraph', or 'Text'.`,
     ),
+    as: PropTypes.string.description(
+      `The DOM tag to use for the element. This the polymorphic as property of
```

Comment:
Can we just drop the `This the polymorphic` ? we don't add this description for other styled-components props.